_examples/rename/rename.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2018 The go-python Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// package rename tests changing the names of methods and functions
+package rename
+
+//gopy:name say_hi
+func SayHi() string {
+	return "hi"
+}
+
+type MyStruct struct {
```

Comment:
couldn't types be renamed as well?

Reply:
It would involve a lot more changes, at the very least renaming would be necessary for generating method receivers/arguments. I'd say that it's not worth the trouble at this point.

Reply:
fair enough :)